### PR TITLE
Fixed exceeded gas error while calling the purchase_in_escrow method

### PR DESCRIPTION
**Pull request by kasodon**

### CODE REVIEW

File: contracts/escrow.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { call, LookupMap, NearBindgen, view, assert, near, UnorderedMap, NearPro
 
 @NearBindgen({})
 export class EscrowContract {
-  GAS_FEE = 30_000_000_000_000; // 30 TGAS
+  // GAS_FEE = 300_000_000_000_000; // 300 TGAS
```
Comment (idea404) [inline contracts/escrow.js]: Why this change if all calls below remain at 30TGas?

**Review by idea404 — CHANGES_REQUESTED** [review]
Thanks for the PR! README section looks very good. Changes in the smart contract code are not required, thanks!

File: contracts/escrow.js (hunk 1 above)
Comment (kasodon) [reply contracts/escrow.js]: I just fixed this and changed the default to 30TGas while repassing the TGas variable. One question I have regarding the parent code is what you were trying to achieve in this [line of code](https://github.com/near-examples/escrow-js/blob/4929ec4c29fd54e9c1cea523940dc922afacd723/contracts/escrow.js#L44)

**Comment by kasodon** [comment]
> Thanks for the PR! README section looks very good. Changes in the smart contract code are not required, thanks!

Thanks!

File: contracts/escrow.js (hunk 1 above)
Comment (idea404) [reply contracts/escrow.js]: add readability to the repo, instead of passing the fixed amount every time
